### ISSUE TITLE
Data of XXXJPY pairs

### ISSUE THREAD

**Issue by hrdrq** [FX-Data/FX-Data-Support#9]
I think the prices are weird.
For example, USDJPY on 2018.06.17 is about `110.6`, but the data is something like `1.10669`.

By the way, what is the format of the csv files?
datetime, bid, ask, volume, volume?

Thank you.

**Comment by kenorb** [comment]
You're right, I think the data needs to be multiplied by 100 for some reason.

The data is generated using a [`fx-data-download.py`](https://github.com/FX31337/FX-BT-Scripts/blob/master/fx-data-download.py) script, e.g.

```
$ ./fx-data-download.py -p EURJPY -y 2018 -m 6 -c
```

I've checked original CSV and the price needs to be multiplied manually.

Similar happens for previous metals, where symbol point needed to be adjusted in the script.
